packages/sdk/src/sampler.ts

```diff
@@ -116,6 +136,8 @@ export class SwSampler implements Sampler {
       traceOptions,
       traceState,
     )
+
+    this.recordMetricsFunction()
```

Comment:
Also makes sense this is called at sampling decision for a counter. [Asking in doc](https://swicloud.atlassian.net/wiki/spaces/NIT/pages/3753116438/AWS+Lambda+Instrumentation+POC?focusedCommentId=3887530249) for my sanity.